pubnub/features.py

```diff
@@ -0,0 +1,20 @@
+from os import getenv
+from pubnub.exceptions import PubNubException
+
+flags = {
+    'PN_ENABLE_ENTITIES': False
```

Comment:
It will be better to set up value of this variable on the runtime, rather than doing it every time when function is called.

Reply:
done